setup.md

```diff
@@ -1,16 +1,16 @@
-# Sovrin -- identity for all
```

Comment:
We should not to forget updating other public documentation after this PR is merged (or create a ticket for update).

Reply:
Created INDY-885 for updating documentation beyond GitHub repositories.